fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -843,6 +843,7 @@ void testLimitForAutoCompleteWithMultipleMatchingFields() throws Exception {
                             false,
                             null,
                             COMPLEX_MULTI_GROUPED_WITH_AUTO_COMPLETE_STORED_FIELDS);
+            assertEquals(-35583570, luceneIndexPlan.planHash());
```

Comment:
Why are all the hashes in this file identical? Are they always the same plan?

Reply:
The hashes are identical for four test cases which seem to have identical queries, but use different data sets and execution properties.

Reply:
Okeydok